daemon/create_test.go

```diff
@@ -0,0 +1,20 @@
+package daemon
+
+import (
+	"testing"
+
+	"github.com/docker/docker/api/types/network"
+	"github.com/stretchr/testify/assert"
+)
+
+// Test case for 35752
+func TestVerifyNetworkingConfig(t *testing.T) {
+	name := "mynet"
+	endpoints := make(map[string]*network.EndpointSettings, 1)
+	endpoints[name] = nil
+	nwConfig := &network.NetworkingConfig{
+		EndpointsConfig: endpoints,
+	}
+	err := verifyNetworkingConfig(nwConfig)
+	assert.EqualError(t, err, "no EndpointSettings for mynet")
```

Comment:
Can you assert true on `errdefs.IsInvalidParameter`?

Reply:
Thanks @cpuguy83  The PR has been updated.